src/plugins/matching-cards/matching-cards-tile-display.js

```diff
@@ -75,29 +87,56 @@ function MatchingCardsTileDisplay({ text, sourceUrl, playbackRange, playMedia })
     }
   };
 
+  const renderMediaControlBar = () => {
+    if (resourceType !== RESOURCE_TYPE.audio && resourceType !== RESOURCE_TYPE.video) {
+      return null;
+    }
+
+    const classes = classNames(
+      'MatchingCardsTileDisplay-mediaControlBar',
+      { 'MatchingCardsTileDisplay-mediaControlBar--alwaysVisible': isTouchDevice() }
```

Comment:
TBD if this should actually always be shown (this also making it easier to see which tile is playing currently).

Reply:
Decision: We show the control bar:
- always on touch devices
- when media is playing inside the tile, on non-touch devices